code/drasil-docLang/lib/Drasil/DocumentLanguage.hs

```diff
@@ -361,10 +361,10 @@ mkSolChSpec si (SCSProg l) =
     map (mkSubSCS si) l
   where
     mkSubSCS :: SystemInformation -> SCSSub -> Section
-    mkSubSCS _ (TMs _ _ [])      = error "There are no Theoretical Models"
+    mkSubSCS _ (TMs _ _ [])      = SSD.thModF Doc.section_ []
```

Comment:
What is the empty list for in the `SSD.thModF` call?

Reply:
It is the empty list that would have otherwise included contents. I added that list because `SSD.thModF` takes in two parameters, an `Idea a` and a list of contents.

```
thModF :: (Idea a) => a -> [Contents] -> Section
thModF progName otherContents = if not (null otherContents) then
                                  SRS.thModel 
                                    (thModIntro progName : otherContents) []
                                else
                                  SRS.thModel 
                                    (thModIntroNoContent progName : 
                                    otherContents) []
```

Reply:
Sorry, what is the list of contents for? Is it ever used right now? It seems to be always empty, no?

Reply:
I believe it usually takes in the field names as its contents in the example to generate the sidebar titles in the table.
![image](https://github.com/JacquesCarette/Drasil/assets/110783820/69d2fb7e-bb73-44fe-9db3-7bea54dba9c3)

For example for `GamePhysics`, it takes in the following parameters to generate the TM:
https://github.com/JacquesCarette/Drasil/blob/2d90402363fb59a38ba062ec93c979ac84e83ff8/code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs#L81

and the `stdFields` are defined further in the file as the following:
https://github.com/JacquesCarette/Drasil/blob/2d90402363fb59a38ba062ec93c979ac84e83ff8/code/drasil-example/gamephysics/lib/Drasil/GamePhysics/Body.hs#L134-L135